test/utils/common.go

```diff
@@ -0,0 +1,32 @@
+package utils
+
+import (
+	"testing"
+)
+
+func TestCommon(t *testing.T, params *TestParams) {
+	if params.TestEnvConfig == true {
+		SetEnvConfig(t, params.Snap, params.App, params.DefaultServicePort)
+	}
+	if params.TestAppConfig == true {
+		SetAppConfig(t, params.Snap, params.App, params.DefaultServicePort)
+	}
+	if params.TestGlobalConfig == true {
+		SetGlobalConfig(t, params.Snap, params.App, params.DefaultServicePort)
+	}
+	if params.TestMixedConfig == true {
+		SetMixedConfig(t, params.Snap, params.App, params.DefaultServicePort)
+	}
+	if params.TestOpenPorts != nil {
+		WaitServiceOnline(t, 60, params.DefaultServicePort)
+	}
```

Comment:
1) Is the array is ever nil? The condition should be `len(params.TestOpenPorts) > 0`.

2) The value isn't used. There are cases where we need to check multiple ports. E.g. https://github.com/canonical/edgex-snap-testing/blob/fcd02aefccc3df92ee26440170f79584ae331d55/test/suites/ekuiper/network_test.go